### PR TITLE
update instructions to use hooks instead of redux

### PR DESCRIPTION
**Pull request by jameyel**
give the project a little facelift while we discuss further updates

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
+- Use [fontawesome](http://fontawesome.io/icons/) for any icon needs and [axios](https://github.com/axios/axios) for any http requests.
+- Spend, at max, 4 hours on this project
+- Prefer to see effective use of state management, thoughtfulness in component building and focus on performance/best practices over styling if you are running out of time
 
 ### Requirements
 
 - Integrate with "TheMealDB" API http://www.themealdb.com/api.php
 - The main page will display 5 random recipes as 'Recipes of the Day'
 - Mobile designs are provided, get creative with responsive desktop designs
 - Use a css preprocessor
```
Comment (zearle) [inline readme.md:16]: Do we want this to be Styled components or something similar?

File: readme.md [hunk 2]
```diff
@@ -28,8 +28,8 @@
 - If any favorited recipes are detected in the browser, add a button on the home page that shows a list of these recipes
```
Comment (zearle) [inline readme.md:35]: How do y'all feel about removing some of the features? One of the issues is that its very difficult to finish this in four hours so instead focusing on important things like the best way to manage state or some of the meatier problems we'd like to see out of a candidate they knock out all the easy stuff, which isn't really indicative of their skill level. I'd prefer to have one or two of these features and also include comments about wanting to see best performance and accessibility practices.

Comment (zearle) [reply readme.md:35]: Maybe something that would take ~2 hours on average but have a maximum of 4 if they really wanted to do something creative or add an extra polish.

File: readme.md (hunk 1 above)
Comment (jameyel) [reply readme.md:16]: wasnt sure if it was as important to limit the css pre-processor as most can pick up the different options pretty quickly if they have an understanding of css or how pre-processors work. didnt want them to spend time learning new libraries if they didnt have to.

File: readme.md (hunk 2 above)
Comment (jameyel) [reply readme.md:35]: agreed. what do you recommend? i'll update the PR to have a couple less requirements then we can see if that's still too much

Comment (jameyel) [reply readme.md:35]: updated.

File: readme.md (hunk 1 above)
Comment (zearle) [reply readme.md:16]: Can we do css preprocessor or JSS? Whichever they feel more comfortable with.

**Pull request merged**